### PR TITLE
Fix #10823

### LINKED ISSUE #10823: Bug (v5-beta1) GitHub Login
I'm using the new v5 beta with the github auth plugin.

Here's our config:
```
[auth.github]
enabled = true
allow_sign_up = true
client_id = ${github_client_id}
client_secret = ${github_client_secret}
scopes = user:email,read:org
auth_url = https://github.com/login/oauth/authorize
token_url = https://github.com/login/oauth/access_token
api_url = https://api.github.com/user
team_ids = ${github_team_id}
allowed_organizations = ${github_team_name}
```

When a user (who belongs to the github organization) attempts to login we receive:

image

I was able to get some info from the logs:

```
t=2018-02-07T14:51:24+0000 lvl=info msg="Request Completed" logger=context userId=0 orgId=0 uname= method=GET path=/login/github status=302 remote_addr=10.x.x.x. time_ms=0 size=304 referer=https://grafana.mydomain.com/login
t=2018-02-07T14:51:25+0000 lvl=eror msg="User not a member of one of the required organizations" logger=context userId=0 orgId=0 uname=
```

### PR DESCRIPTION
**Pull request by inish777**
I've changed Github organizations api url.

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -631,6 +696,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "4de68f1342ba98a637ec8ca7496aeeae2021bf9e4c7c80db7924e14709151a62"
+  inputs-digest = "b643d3f62f346625b8be7e41b99e719b5d9d4c24f82141f1af5c01b20d2bac7c"
```
Comment (torkelo) [inline Gopkg.lock]: can you remove this file from the PR? you must have run dep?

Comment (inish777) [reply Gopkg.lock]: @torkelo oh, sure, one moment

**Comment by torkelo** [comment]
does this work both for public and private github orgs?

**Comment by torkelo** [comment]
I guess we have to revert this since this seems to break the feature. https://github.com/grafana/grafana/issues/10964

**Pull request merged**